### ISSUE TITLE
can not add new properties to products

### ISSUE THREAD

**Issue by mehdisafavi** [OFFLINE-GmbH/oc-mall-plugin#130]
when i try to create a new property for category , it don,t appears in product update page tabs . but when i check it by old property group (seed) data it work's . how we can create new property group for a product? .

**Comment by mehdisafavi** [comment]
Yes . when i check  it on https://mall.offline.swiss , i faced same issue . i use mall version 1.0.18   .

**Comment by tobias-kuendig** [comment]
Can you upgrade to the latest version and try again? This might be related to a bug introduced in version 1.0.18

**Comment by mehdisafavi** [comment]
Solved by upgrading . Thank You .
